static/app/components/feedback/feedbackItem/feedbackItem.tsx

```diff
@@ -122,6 +132,34 @@ export default function FeedbackItem({feedbackItem, eventData, tags}: Props) {
   );
 }
 
+function TraceDataSection({
```

Comment:
can we move this to a new file? similar to how the other sections are structured

Reply:
hm I feel like it's small enough to keep here, it's basically a wrapper around Issue's TraceDataSection